frontend/kesaseteli/employer/src/pages/_app.tsx

```diff
@@ -44,6 +45,7 @@ const App: React.FC<AppProps> = ({ Component, pageProps }) => (
           </Layout>
         </ThemeProvider>
       </AuthProvider>
+      <HiddenLoadingIndicator />
```

Comment:
Also added this hidden loading indicator to the _app for helping the browser tests to check that all backend requests are done